OmiseSDK/Source.swift

```diff
@@ -52,9 +53,12 @@ public struct CreateSourceParameter: Encodable {
     /// The currench of the creating Source
     public let currency: Currency
     
+    public let platformType: String
```

Comment:
I'm not sure what this is used for, but it sounds more like something internal? So maybe better make it `private`?